tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -74,10 +88,16 @@ spec:
           repository=$(jq -r '.repository' <<< $component)
           name=$(jq -r '.name' <<< $component)
           git_sha=$(jq -r '.source.git.revision' <<< $component) # this sets the value to "null" if it doesn't exist
-          # take tag if present in mapping file, otherwise take ReleaseStrategy default provided as a task param
-          tag=$(params.tag)
-          if [ $(jq 'has("tag")' <<< $component) == "true" ] ; then
-              tag=$(jq -r '.tag' <<< $component)
+          # prefixedTag should be used when tagPrefix parameter length is non-zero
+          # In case it is zero, it take tag if present in mapping file, otherwise take ReleaseStrategy default provided
+          # as a task param
+          if [ $prefixedTag ] ; then
+              tag=$prefixedTag
+          else
+              tag=$(params.tag)
```

Comment:
These 4 lines can be simplified to a single `jq` command:
`tag=$(jq -r '.tag // "$(params.tag)"' <<< $component)`

Reply:
changed.